test/integration/element-test.js

```diff
@@ -24,22 +24,19 @@ suite('element methods', function() {
   });
 
   test('#findElement', function() {
-    var root = client.findElement('html');
-    assert.ok(root.findElement('body').displayed());
+    client.findElement('html');
```

Comment:
Could we at least check to make sure that the thing returned from `findElement` looks like what we're expecting? I don't love tests without assertions -- it makes it unclear what's being tested.